exercises/change/metadata.yml

```diff
@@ -1,5 +1,2 @@
 ---
 blurb: "Correctly determine change to be given using the least number of coins"
-source: "Software Craftsmanship - Kata-logue"
-source_url: "http://craftsmanship.sv.cmu.edu/exercises/coin-change-kata"
-
```

Comment:
Is not having a `source_url` or `source` allowed by Exercism?  That won't break anything?  Would it be enough to just remove `source_url`?

Perhaps remove `source_url` and update `source` to indicate no url available?

Reply:
It appears to be ok:

https://github.com/exercism/problem-specifications/blob/master/exercises/all-your-base/metadata.yml
https://github.com/exercism/problem-specifications/blob/master/exercises/alphametics/metadata.yml
https://github.com/exercism/problem-specifications/blob/master/exercises/bank-account/metadata.yml

> Perhaps remove source_url and update source to indicate no url available?

I thought about this, but searching online for `Software Craftsmanship - Kata-logue` doesn't turn up anything helpful, and is almost misleading